GOALS

```diff
@@ -0,0 +1,7 @@
+gitsh hits 1.0 when:
+
+[x] it provides a useful interactive shell environment.
+[x] it runs scripts.
+[ ] it has the kinds of features expected by advanced shell users.
```

Comment:
This seems a bit vague. Should this have examples?

Reply:
I was thinking of these features: #321, #325, #145, #100, #67, and maybe also #303, #41, #35

Not sure if it's redundant to list them out here and in GitHub issues? I could also imagine someone suggesting a very reasonable feature that fits in this list and seems like it should be in 1.0, but maybe I should resist that kind of feature creep.